packages/bpk-docs/src/components/neo/PageHead/PageHead.js

```diff
@@ -56,9 +56,9 @@ const PageHead = (props: Props) => {
         <BpkText
           bold
           tagName="h2"
-          className={getClassName('bpkdocs-page-head__available-configs')}
+          className={getClassName('bpkdocs-page-head__in-section')}
         >
-          Available configurations
+          In this section:
```

Comment:
I don't think this should end in a colon. It's a lead-in statement to a list, but it isn't a complete sentence on its own, which, as a rule of thumb, dictates when the colon should appear.

https://www.quickanddirtytips.com/education/grammar/formatting-vertical-lists?page=2#introductory-colons